app/templates/login.html

```diff
@@ -25,17 +36,44 @@
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
 
-  {% if error %}
-    {% with lede = error %}
-      {% include "toolkit/forms/validation.html" %}
-    {% endwith %}
-  {% endif %}
-  <form autocomplete="off" action="{{ url_for('.login') }}" method="post" class="question">
-    <div style="display:none;"><input id="csrf_token" name="csrf_token" type="hidden" value="{{ csrf_token() }}"></div>
-    <label class="question-heading" for="username">Username</label>
-    <input autocomplete="off" type="text" name="username" id="username" class="text-box" value="{{previous_responses.username}}">
-    <label class="question-heading" for="password">Password</label>
-    <input autocomplete="off" type="password" name="password" id="password" class="text-box">
+  <form autocomplete="off" action="{{ url_for('.render_login') }}" method="post">
```

Comment:
Or if you leave the form action empty here then I think it might post including the GET parameters.  But I'm not 100% sure that will work.